### PR TITLE
Components v2 loader

### LINKED ISSUE #963: Components v2: Loader
copy from old components

### PR DESCRIPTION
**Pull request by keip**
# Props
same as old props

resolves #963

### CODE REVIEW

File: packages/components-storybook-v2/.storybook/config.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ addParameters({
     options: {
         theme: Theme,
         showPanel: true,
-        panelPosition: 'bottom',
+        panelPosition: 'right',
```
Comment (slowbackspace) [inline packages/components-storybook-v2/.storybook/config.js:10]: why this change?

File: packages/components-storybook-v2/test/integration/Loaders.test.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+describe('Icons', () => {
```
Comment (slowbackspace) [inline packages/components-storybook-v2/test/integration/Loaders.test.js]: Icons? Probably Loader right?

File: packages/components-storybook-v2/.storybook/config.js (hunk 1 above)
Comment (keip) [reply packages/components-storybook-v2/.storybook/config.js:10]: i like the addon pannel on the right side, it's more visible and clearer...

File: packages/components-storybook-v2/test/integration/Loaders.test.js (hunk 2 above)
Comment (keip) [reply packages/components-storybook-v2/test/integration/Loaders.test.js]: yup it should be Loader. fixed it, need to update snapshots. adding in progress label...

**Comment by slowbackspace** [comment]
just an idea for future generations. Couldn't we place the text under the loader rather than inside it? our 'design' doesn't really work so well.

Screenshot 2019-12-30 at 10 17 41

**Review by slowbackspace — APPROVED** [review]
